service-statistics-unsd/src/main/java/fi/nls/oskari/control/statistics/plugins/unsd/requests/UnsdRequest.java

```diff
@@ -0,0 +1,159 @@
+package fi.nls.oskari.control.statistics.plugins.unsd.requests;
+
+import fi.nls.oskari.control.statistics.data.StatisticalIndicatorDataModel;
+import fi.nls.oskari.control.statistics.plugins.APIException;
+import fi.nls.oskari.control.statistics.plugins.unsd.UnsdConfig;
+import fi.nls.oskari.util.IOHelper;
+
+import java.net.HttpURLConnection;
+import java.util.*;
+
+/**
+ * Base class for UNSD statistics queries.
+ */
+public class UnsdRequest {
+
+    private static final String PLACEHOLDER_GOAL = "{goalCode}";
+    private static final String PARAM_INDICATOR = "indicator";
+    private static final String PARAM_PAGE = "page";
+    private static final String PATH_GOAL = "Goal/" + PLACEHOLDER_GOAL;
+    private static final String PATH_TARGETS = PATH_GOAL + "/Target/List";
```

Comment:
I'd remove `PATH_GOAL` and replace `PATH_TARGETS` and `PATH_DIMENSIONS` with
`private static final String PATH_TARGETS = "Goal/{goalCode}/Target/List";`
`private static final String PATH_DIMENSIONS = "Goal/{goalCode}/Dimensions";`
to better distinct PARAM_ and PATH_ fields and to make it more clear that the PATH_s must not begin with a `/`